config/packages/framework.yaml

```diff
@@ -1,16 +1,24 @@
 # see https://symfony.com/doc/current/reference/configuration/framework.html
 framework:
     secret: '%env(APP_SECRET)%'
-    default_locale: '%locale%'
     #csrf_protection: true
-    #http_method_override: true
+    http_method_override: false
 
     # Enables session support. Note that the session will ONLY be started if you read or write from it.
     # Remove or comment this section to explicitly disable session support.
     session:
-        enabled: false
+        handler_id: null
+        cookie_secure: auto
+        cookie_samesite: lax
+        storage_factory_id: session.storage.factory.native
```

Comment:
why this is back? This was gone somewhere around switching to JWT. Are you sure that it won't enable sessions back?

Reply:
Flex recipe... reverted